browser/ui/views/frame/brave_browser_frame_mac.mm

```diff
@@ -43,3 +46,20 @@
   BrowserFrameMac::GetWindowFrameTitlebarHeight(override_titlebar_height,
                                                 titlebar_height);
 }
+
+void BraveBrowserFrameMac::ValidateUserInterfaceItem(
+    int32_t tag,
+    remote_cocoa::mojom::ValidateUserInterfaceItemResult* result) {
+  BrowserFrameMac::ValidateUserInterfaceItem(tag, result);
+
+  if (tag != IDC_TOGGLE_TAB_MUTE || !result->enable) {
+    return;
+  }
+
+  // Update toggle state for tab mute menu bar entry.
+  Browser* browser = browser_view_->browser();
```

Comment:
did you check that it works fine when there's no open windows at all?

Reply:
Yes, it's ok because this `Frame` instance only exists with browser window.